zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5765,6 +5765,45 @@ components:
             As a result, the completion request is rejected and the task remains active.
             Defaults to false.
           nullable: true
+        corrections:
+          $ref: "#/components/schemas/JobResultCorrections"
+    JobResultCorrections:
+      type: object
+      description: Attributes that were corrected by the worker. Others are considered not corrected.
+      nullable: true
+      properties:
+        assignee:
+          type: string
+          description: Assignee of the task.
+        dueDate:
+          type: string
+          format: date-time
+          description: The due date of the task.
+          nullable: true
+        followUpDate:
+          type: string
+          format: date-time
+          description: The follow-up date of the task.
+          nullable: true
+        candidateUsersList:
+          type: array
+          description: The list of candidate users of the task.
+          items:
+            type: string
+          nullable: true
+        candidateGroupsList:
+          type: array
+          description: The list of candidate groups of the task.
+          items:
+            type: string
+          nullable: true
```

Comment:
🔧 With [`UserTaskChangeset`](https://github.com/camunda/camunda/blob/9cb4ef558a5ed06391678a7af0a63af2cc01fb22/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L5519-L5536) in mind, we should actually refer to `candidateUsers` and `candidateGroups` rather the `*List` variants. I had not considered this, when I [said here](https://github.com/camunda/camunda/pull/25663#issuecomment-2536082943) that we should use it in the API. I'd rather have consistency in the API first.